modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc

```diff
@@ -0,0 +1,20 @@
+// This module is included in the following assembly:
+//
+// * installing/installing-openshift-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc_{context}"]
+= Disabling ShipwrightBuilds build controller and webhook pod
+
+You can disable the `ShipwrightBuilds` Custom Resource (CR) without disabling the shared-resources Container Storage Interface (CSI) driver component. Disabling `ShipwrightBuilds` CR removes the `ShipwrightBuilds` build controller and `ShipwrightBuilds` build webhook pod from the `openshift-builds` namespace.
+
+.Procedure
+
+. Disable the ShipwrightBuilds controller and webhook by running the following command:
++
+[source,terminal]
+----
+oc patch openshiftbuild cluster --type=merge -p '{"spec":{"shipwright":{"build":{"state":"Disabled"}}}}'
+----
+
+This command disables the build controller and webhook but does not affect the ShipwrightBuilds Custom Resource Definition (CRD).
```

Comment:
Add a plus (+) symbol to indent this paragraph with the procedure step.

Reply:
ShipwrightBuilds > `ShipwrightBuilds`

A CR or a CRD? `ShipwrightBuilds` CR is mentioned at the intro of the module but it's CRD in this part.

Reply:
AFAIK, A CRD is for creating custom resources types. A CR is for creating a resource based on the resource type defined in a CRD.